cmd/root.go

```diff
@@ -87,6 +88,7 @@ func startServer() (func() error, func(err error)) {
 				a.MountRouter("ListenBrainz Auth", consts.URLPathNativeAPI+"/listenbrainz", CreateListenBrainzRouter())
 			}
 			if conf.Server.Prometheus.Enabled {
+				go core.MetricsWorker()
```

Comment:
To have a "sub-processe" running in background, it should be started in the same way as the others, using the `errgroup` in  the `runNavidrome()` method.  (Please rebase as the `root.go` changed a bit)